extension/content/personas.css

```diff
@@ -90,6 +90,20 @@
   font-weight: bold !important;
 }
 
+#personas-toolbar-button.seamonkey, #personas-toolbar-button.seamonkey:hover, #personas-toolbar-button.seamonkey:hover:active {
+  list-style-image: url("chrome://personas/content/personas_30x30.png");
+  -moz-image-region: rect(0, 30px, 30px, 0px);  
+}
+
+toolbar[iconsize="small"] #personas-toolbar-button.seamonkey, toolbar[iconsize="small"] #personas-toolbar-button.seamonkey:hover, toolbar[iconsize="small"] #personas-toolbar-button.seamonkey:hover:active {
+  list-style-image: url("chrome://personas/content/personas_20x20.png");
+}
+
+#personas-menu.seamonkey {
```

Comment:
So, the rules in 103 and 104 are the same as 48 and 49. So, instead of re-declaring them here, just add the selector `#personas-menu.seamonkey` to line 46/47 like this:
```
#personas-selector-button,
#personas-toolbar-button,
#personas-menu.seamonkey {
```